library/ceph_pool.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python
+#coding: utf-8 -*-
+
+DOCUMENTATION = """
+---
+author: Michael Sambol
+module: ceph_pool
+short_description: Creates ceph pool and ensures correct pg count
+description:
+  There are three possible outcomes:
+    1/ Create a new pool if it doesn't exist
+    2/ Set the pool's pg count to correct number
+    3/ Nothing: pool is created and pg count is correct
+options:
+  pool_name:
+    description:
+      - The pool in question: create it or ensure correct pg count
+    required: true
+  constant:
+    description:
+      - A constant multiplier used in calculating pg count
+    required: true
+  copies:
+    description:
+      - The number of copies the cluster keeps
+    required: true
```

Comment:
Well, I'm trying to look at the module without the context of how you'll be using it. I'm reviewing the module as a stand alone piece of work.

Perhaps the module should error if the supplied copies argument doesn't equal the number of copies a pool already has?